skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,316 @@
+import numpy as np
+import scipy as sp
+
+from .._shared.utils import check_nD
+
+
+class TpsTransform:
+    """Thin-plate splines transformation.
```

Comment:
```suggestion
    """Thin-plate spline transformation.
```
I think we don't want the final -s because the noun is used as adjective here.